modular_skyrat/modules/GAGS/json_configs/costumes/misc.json

```diff
@@ -6,5 +6,31 @@
 			"blend_mode": "overlay",
 			"color_ids": [ 1 ]
 		}
+	],
```

Comment:
TIL, the gags .jsons scream if you have two states with different amounts of color_ids. I'll need to make a unique .json for this damn vest.